feature/experimental/isis/ate_tests/lsp_updates_test/lsp_updates_test.go

```diff
@@ -40,8 +40,13 @@ func TestOverloadBit(t *testing.T) {
 	overloads := isisPath.Level(2).SystemLevelCounters().DatabaseOverloads()
 	setBit := isisPath.Global().LspBit().OverloadBit().SetBit()
 	deadline := time.Now().Add(time.Second)
+	checkSetBit := check.Equal(setBit.State(), false)
+	if *deviations.AllowNilForDefaults {
```

Comment:
Can we change the deviation flag to MissingValueForDefaults? I think "allow" is not really accurate in the name because it's not the device allowing nil, but rather the test is allowing it. We generally want the deviation to describe the device behavior, not the test behavior. I also commented on #638.

Reply:
Fixed. I also removed the `MixedOcCliPrefersCli` deviation, which was added accidentally due to a rebasing mistake.